File: trunk/Phanfare.ExternalAPI/PhanfareException.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Phanfare.ExternalAPI
{
	public class PhanfareException : ApplicationException
	{
		public readonly int ErrorCode;
		public readonly string ErrorName;
		
		public PhanfareException( string message )
			: base( message )
		{
		}

		public PhanfareException( int errorCode, string errorName, string message )
			: base( message )
		{
			this.ErrorCode = errorCode;
			this.ErrorName = errorName;
		}
	}
}

File: trunk/PhanfareJavaAPI/src/com/phanfare/api/PhanfareException.java
package com.phanfare.api;

public class PhanfareException extends Exception {
	private static final long serialVersionUID = 1L;
	public int errorCode;
	public String errorValue;

	public PhanfareException(int errorCode, String errorValue, String message) {
		super(message);
		this.errorCode = errorCode;
		this.errorValue = errorValue;
	}
}
